src/SDK.ts

```diff
@@ -624,14 +624,14 @@ export default class SDK implements ISDK {
     });
   }
 
-  /**
+  /**'
    * Starts a session to omnichannel.
    * @param requestId: RequestId to use for session init.
    * @param sessionInitOptionalParams: Optional parameters for session init.
    */
-  public async sessionInit(requestId: string, sessionInitOptionalParams: ISessionInitOptionalParams = {}): Promise<void> {
+  public async sessionInit(requestId: string, sessionInitOptionalParams: ISessionInitOptionalParams = {}, useSessionInitV2 = false): Promise<void | FetchChatTokenResponse> {
```

Comment:
why is not part of the optionalParams?, it will be cleaner

Reply:
👆